be/src/runtime/routine_load/data_consumer.cpp

```diff
@@ -82,7 +81,7 @@ Status KafkaDataConsumer::init(StreamLoadContext* ctx) {
 
     RETURN_IF_ERROR(set_conf("metadata.broker.list", ctx->kafka_info->brokers));
     RETURN_IF_ERROR(set_conf("group.id", group_id));
-    RETURN_IF_ERROR(set_conf("enable.partition.eof", "false"));
+    RETURN_IF_ERROR(set_conf("enable.partition.eof", "true"));
```

Comment:
Could you add more comments for what it stands for. and why is it set to true?

Reply:
Fixed